### ISSUE TITLE
data.tables are returned silently

### ISSUE THREAD

**Issue by sbfnk** [epiforecasts/ringbp#64]
E.g. running the readme followed by `res` will not produce any output until `res` is called again.

This can be fixed by returning `res[]` instead of `res` e.g. in  https://github.com/epiforecasts/ringbp/blob/972bf86544e716b52d0da9f60f4681372760e9ec/R/scenario_sim.R#L77

**Comment by joshwlambert** [comment]
@sbfnk this issue can now be closed after #80 was merged. I linked the issue in the  commit message which should have automatically closed it, not sure if it's due to the PR from a fork or the squash merge, or something else.

**Comment by sbfnk** [comment]
Fascinating, this looks like a github bug to me tbh - hovering over the (i) above yields a message that the issue should close once the commit has been merged into main.
